as-controller-board-status/functions/output.go

```diff
@@ -246,15 +248,49 @@ func (boardStatus *CheckBoardStatus) processVendingDoorState(lc logger.LoggingCl
 			return fmt.Errorf("failed to submit the controller board's status to the central vending state service: %v", err.Error())
 		}
 
-		// Prepare a message to be sent to the MQTT bus. Depending on the state
-		// of the door, this message may trigger a CV inference
-		err = boardStatus.RESTCommandJSON(boardStatus.Configuration.DoorStatusCommandEndpoint, http.MethodPut, VendingDoorStatus{
-			VendingDoorStatus: strconv.FormatBool(doorClosed),
-		})
+		// Prepare and send EdgeX command. Depending on the state of the door, this message may trigger a CV inference
+		settings := make(map[string]string)
+		settings["VendingDoorStatus"] = strconv.FormatBool(doorClosed)
+		err = boardStatus.SendCommand(lc, http.MethodPut, boardStatus.Configuration.InferenceDeviceName, boardStatus.Configuration.InferenceDoorStatusCmd,
+			settings)
 		if err != nil {
-			return fmt.Errorf("failed to submit the vending door state to the MQTT device service: %v", err.Error())
+			return fmt.Errorf("failed to submit the vending door state to the command client: %v", err.Error())
 		}
 	}
 
 	return nil
 }
+
+// SendCommand issues CommandClient GET and SET command calls, CommandClient takes care of http calls,
+// here the requirement are actionName, deviceName, commandName and settings, logger client is needed for logging
+func (boardStatus *CheckBoardStatus) SendCommand(lc logger.LoggingClient, actionName string, deviceName string,
+	commandName string, settings map[string]string) error {
+	lc.Debug("Sending Command")
+
+	switch actionName {
+	case http.MethodPut:
+		lc.Infof("executing %s action", actionName)
+		lc.Infof("Issuing SET command '%s' for device '%s'", commandName, deviceName)
```

Comment:
These and below should be `Debugf`. `Info` should only be used during service start-up code.

Reply:
Then https://github.com/intel-retail/automated-vending/blob/23a2dfdf004f6b778a25450018a4dec5eb9cf73c/as-vending/functions/output.go#L443 needs to be changed too?

Reply:
Yes.